applications/GeoMechanicsApplication/custom_strategies/schemes/geomechanics_time_integration_scheme.hpp

```diff
@@ -331,14 +325,16 @@ class GeoMechanicsTimeIntegrationScheme : public Scheme<TSparseSpace, TDenseSpac
             << rModelPart.GetBufferSize() << std::endl;
     }
 
-    void CheckSolutionStepsData(const Node& r_node, const Variable<double>& variable) const
+    template <class T>
+    void CheckSolutionStepsData(const Node& r_node, const Variable<T>& variable) const
     {
         KRATOS_ERROR_IF_NOT(r_node.SolutionStepsDataHas(variable))
             << variable.Name() << " variable is not allocated for node "
             << r_node.Id() << std::endl;
     }
 
-    void CheckDof(const Node& r_node, const Variable<double>& variable) const
+    template <class T>
+    void CheckDof(const Node& r_node, const Variable<T>& variable) const
```

Comment:
And the same style issue is found here:
```suggestion
    void CheckDof(const Node& rNode, const Variable<T>& rVariable) const
```

Reply:
Done